nemo/collections/asr/models/ctc_models.py

```diff
@@ -334,9 +346,12 @@ def training_step(self, batch, batch_nb):
 
         if (batch_nb + 1) % row_log_interval == 0:
             wer_num, wer_denom = self._wer(predictions, transcript, transcript_len)
-            tensorboard_logs.update(
-                {'training_batch_wer': wer_num / wer_denom,}
-            )
+            tensorboard_logs.update({'training_batch_wer': wer_num / wer_denom})
+
+            pred_text = self._wer.ctc_decoder_predictions_tensor(predictions[:1])
```

Comment:
should we worry about an overhead of this?
Maybe let's at least bump row_log_interval in ASR configs to be way more than 1, (50?)

Reply:
Row log is kinda dependent on how much data is currently there, say you row log number is higher than number of batches, then the script would never log wer at all. (it's only a concern for toy datasets though). Let's take a middle ground of 10 for now.

Reply:
On second thought, 50 was old nemo's default and no one complained so lets just use 50.

Reply:
Done, updated all of the configs for ASR